enhancements/aiops/20230120-Identify-prometheus-metrics-that-match-known-anomaly-chart-patterns.md

```diff
@@ -0,0 +1,93 @@
+# Title: 
+Identify prometheus metrics that match known anomaly chart patterns
```

Comment:
Remove "prometheus" as we are moving to open telemetry metrics

Reply:
done